generator/.DevConfigs/ffdb2735-52f2-4545-9168-eb24bf72874c.json

```diff
@@ -0,0 +1,9 @@
+{
+  "core": {
+    "changeLogMessages": [
+      "Disable signing the payload when UnsignedPayload is set to true"
```

Comment:
It isn't completely clear what `UnsignedPayload` this is now because the property was removed. Unless this is still referencing the trait in the model. This should probably be clarified:

```
Disable signing the payload when the operation is modeled as having an UnsignedPayload set to true.
```

Or

```
Disable signing the payload when the operation is modeled as having an unsigned payload.
```